generator/generator_tool/generator_tool.cpp

```diff
@@ -50,6 +50,20 @@
 
 #include "3party/gflags/src/gflags/gflags.h"
 
+namespace
+{
+std::string GetDataPathText() noexcept
+{
+  return std::string(
```

Comment:
```c++
char const * GetDataPathHelp()
{
  static std::string const kHelp = "...";
  return kHelp.c_str();
}
```
And please, remove noexcept. This is not true. And get rid of gDataPathText (BTW, the right naming must be "g_dataPathText").

Reply:
> And please, remove noexcept.

Done

>  And get rid of gDataPathTex

Done